### PR TITLE
Newsletter for December

### PR DESCRIPTION
**Pull request by bethaitman**

### CODE REVIEW

File: docs/blog/newsletter-december-2021.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+The article's problems with Git - in particular, that it's hard to learn - aren't just a problem for writers, but are complained about by many developers too. There is a counter-argument to needing a new system: instead, we should take the tools we have, but make them work better for use cases. Perhaps we don't need a new version control system, but better Git clients designed for docs workflows…
+
+But some folks disagreed with the premise that standardisation will help. Not least because creating new standards `can make things worse <https://xkcd.com/927/>`__! And how can we standardise if we don't have any universal best practices? Perhaps, for docs problems, there is no "one best way" - you'll always have to adapt your tools to different use cases; sometimes what you really need is the feature set of a Help Authoring Tool like Flare. Or perhaps we should all just give up and use Word? (joking, joking!)
```
Comment (hillaryfraley) [inline docs/blog/newsletter-december-2021.rst:62]: hahah good one :D

**Review by hillaryfraley — APPROVED** [review]
👍  looks great!

File: docs/blog/newsletter-december-2021.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+One last call for folks who haven’t yet filled out the `Salary Survey <https://salary-survey.writethedocs.org/>`__. We’d love it if you would consider contributing the details of your compensation. It’s all anonymous and the anonymised results are released for free, so everyone in the community can benefit. `Fill it out here <https://salary-survey.writethedocs.org/>`__.
+
+With that, on to our articles from this month!
+
+--------------------------------------------------------------
+The work documentation really requires (it's not all writing!)
+--------------------------------------------------------------
+
+A plaintive question about how to develop content more efficiently produced a perhaps surprising set of responses. The original poster framed their challenge in terms of research that delays the writing process, but replies tended to point out that research is a key part of creating any documentation. If your research is inefficient -- if you're falling down "useless rabbit holes" as one poster put it -- you could consider pruning your sources, or working more from summaries than from long form reads.
```
Comment (plaindocs) [inline docs/blog/newsletter-december-2021.rst]:
```suggestion
A plaintive question about how to develop content more efficiently produced a perhaps surprising set of responses. The original poster framed their challenge in terms of research that delays the writing process, but replies tended to point out that research is a key part of creating any documentation. If your research is inefficient -- if you're falling down "useless rabbit holes" as one poster put it -- you could consider pruning your sources, or working more from summaries than from long-form reads.
```

File: docs/blog/newsletter-december-2021.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+
+Generally, though, folks agreed that research and other supporting activities form the bulk of the work they do to produce docs -- actual writing takes up relatively little time overall. In addition to research, testing/QA, wrangling reviews and tooling, managing content beyond the writing (structure, frontmatter, more) all showed up on lists of docs work beyond writing. So maybe when we're not writing we're still as efficient as we can be?
```
Comment (plaindocs) [inline docs/blog/newsletter-december-2021.rst]:
```suggestion
Generally, though, folks agreed that research and other supporting activities form the bulk of the work they do to produce docs -- actual writing takes up relatively little time overall. In addition to research, testing/QA, wrangling reviews and tooling, managing content beyond the writing (structure, metdata, etc) all showed up on lists of docs work beyond writing. So maybe when we're not writing we're still as efficient as we can be?
```

File: docs/blog/newsletter-december-2021.rst [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+Documentation rendering was contentious, but folks saw more promise in the markup idea. DITA is heavyweight; Markdown lets you get on with writing, but is neither standardised nor designed for docs; and the alternatives aren't widely used. Some community members defended AsciiDoc as meeting all the criteria - it's interesting that it's currently going through `a standardization process via Eclipse <https://projects.eclipse.org/projects/asciidoc.asciidoc-lang>`__.
+
+The article's problems with Git - in particular, that it's hard to learn - aren't just a problem for writers, but are complained about by many developers too. There is a counter-argument to needing a new system: instead, we should take the tools we have, but make them work better for use cases. Perhaps we don't need a new version control system, but better Git clients designed for docs workflows…
```
Comment (plaindocs) [inline docs/blog/newsletter-december-2021.rst]: > The article's problems with Git - in particular, that it's hard to learn - aren't just a problem for writers, but are complained about by many developers too. 

This reads kinda awkward, but I don't have a great rewording. How about

> Many writers and developers complain about git, most often that it is hard to learn.

File: docs/blog/newsletter-december-2021.rst [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+A short read: Over at Tech Crunch, `Why DEI programs are failing <https://techcrunch.com/2021/11/16/why-dei-programs-are-failing/>`__ discusses the shortcomings of DEI programs and how ERGs might be the answer.
+
+A longer read: A little old, but still relevant, last year’s `McKinsey and Company's 2020 report on Diversity and Inclusion <https://www.mckinsey.com/featured-insights/diversity-and-inclusion/diversity-wins-how-inclusion-matters>`__ concludes that companies investing in DE&I are more profitable than those that are not. You can download the full report at the end of the summary.
+
+For a longer time investment: Check out `Diversity Inc Best Practices' Webinar Recap: Let's talk Microagressions <https://www.diversityincbestpractices.com/webinar-recap-lets-talk-microaggressions/>`__. In this webinar, the panelists define microaggressions, discuss when they first encountered them, how to respond to them, and what their companies are doing to combat them.
+
+.. ----------------
```
Comment (plaindocs) [inline docs/blog/newsletter-december-2021.rst]: This probably all needs unindenting?

**Review by plaindocs — APPROVED** [review]
Looks good to me. 

I wish there was a better way to preview these, due to the published date. 

@mxsasha @ericholscher  do you know if there is a setting on ablog to show future dated posts if we're on a preview somehow?

**Comment by mxsasha** [comment]
> Looks good to me.
> 
> I wish there was a better way to preview these, due to the published date.
> 
> @mxsasha @ericholscher do you know if there is a setting on ablog to show future dated posts if we're on a preview somehow?

[Doesn't seem like it](https://ablog.readthedocs.io/en/latest/manual/ablog-configuration-options/) but we may be able to build a [draft archive page](https://ablog.readthedocs.io/en/latest/manual/posting-and-listing/#directive-post).

File: docs/blog/newsletter-december-2021.rst (hunk 4 above)
Comment (hzoppetti) [reply docs/blog/newsletter-december-2021.rst]: How about: The article's main problem with Git, often expressed by writers as well as developers, is that it is hard to learn.

Comment (plaindocs) [reply docs/blog/newsletter-december-2021.rst]: Better for sure

**Comment by plaindocs** [comment]
@mxsasha yeah, and as we don't commit drafts to master, that should just work?

**Comment by ericholscher** [comment]
It builds the page, you just need to navigate to it directly: https://writethedocs-www--1630.org.readthedocs.build/blog/newsletter-december-2021/

File: docs/blog/newsletter-december-2021.rst (hunk 5 above)
Comment (bethaitman) [reply docs/blog/newsletter-december-2021.rst]: Nope, because no sponsor. I'll remove.

File: docs/blog/newsletter-december-2021.rst (hunk 4 above)
Comment (bethaitman) [reply docs/blog/newsletter-december-2021.rst]: Much better, thanks!

**Pull request merged**